com.github.slgobinath.SafeEyes.yaml

```diff
@@ -0,0 +1,99 @@
+id: com.github.slgobinath.SafeEyes
```

Comment:
this also doesn't match. The app id has to be changed everywhere.

Reply:
Yeah, I'm going to push the commit that changes them. But currently I couldn't figure out one thing. 

So, the app, after setup, installs icons and desktop file under `/app/lib/python3.10/site-packages/safeeyes/platform/` I couldn't find any docs or different packages related to this. How do you think I should proceed? `install`, mv, cp or symlinks don't feel 'clean'